charts/platform-storage/templates/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.admissionController.app_name }}
+  namespace: platform
```

Comment:
let's not define namespace, helm by default installs resources in the namespace where helm release is created

Reply:
done